src/wallet/wallet.h

```diff
@@ -1123,6 +1123,8 @@ class CWallet final : public CCryptoKeyStore, private interfaces::Chain::Notific
 
     std::set<CTxDestination> GetLabelAddresses(const std::string& label) const;
 
+    bool GetNewAddress(const OutputType type, const std::string label, CTxDestination& dest, std::string& error);
```

Comment:
2010b1a13e6b7eaa681cc62ae58079ccbec72816

This should be atomic or add `EXCLUSIVE_LOCKS_REQUIRED(cs_wallet)` annotation?

Also, could make `GetKeyFromPool` private?

Reply:
Done. Also fixed typo